packages/mediacenter/kodi/system.d/kodi.service

```diff
@@ -9,7 +9,7 @@ EnvironmentFile=/usr/lib/kodi/kodi.conf
 EnvironmentFile=-/run/libreelec/kodi.conf
 EnvironmentFile=-/run/libreelec/debug/kodi.conf
 ExecStartPre=-/usr/lib/kodi/kodi-config
-ExecStart=/usr/lib/kodi/kodi.sh --standalone -fs $KODI_ARGS $KODI_DEBUG
+ExecStart=/usr/lib/kodi/kodi.sh --standalone -fs $KODI_AUDIO_ARGS $KODI_ARGS $KODI_DEBUG
```

Comment:
Why not just set the Arg directly here instead of using an ENV variable?

Reply:
because this way it can easily be changed via .config/kodi.conf, including starting kodi with no audio backend arg at all as well

Reply:
Ok, just seems like an extra level of translation that isn't needed.